internal/metric/smart_metrics.go

```
@@ -0,0 +1,154 @@
+		section = section[:endIdx]
+	}
+
+	data := SmartData{}
+	var errors []CustomErr
+	lines := strings.Split(section, "\n")
+
+	for _, line := range lines {
+		if strings.TrimSpace(line) == "" || strings.HasPrefix(line, "===") {
+			continue
+		}
+		
```

golangci-lint (1.62.2)

File is not `goimports`-ed

(goimports)